### ISSUE TITLE
set `--change-id` in ci

### ISSUE THREAD

**Issue by jyn514** [rust-lang/rust#119296]
right now, every ci job starts with this warning:
```
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 119124` at the top of `config.toml`
```
this is silly. it's ci. nobody is configuring different defaults. we should always silence this.

unfortunately this is somewhat tricky now that change-id is no longer stored in config.example.toml. maybe we can set `./configure --set change-id=${github.event.number}` in `src/ci/run.sh`? i am not sure how the change tracker handles a PR number it doesn't recognize.

cc @onur-ozkan

**Comment by onur-ozkan** [comment]
2 options come to my mind:

- Print only if it's not a CI (although this might not be a good idea as it will affect any CI pipeline, including package releases from Linux distributions).
- Set `change-id` to very large value (like 9999999) in run.sh

**Comment by onur-ozkan** [comment]
> i am not sure how the change tracker handles a PR number it doesn't recognize.

If the change-id is greater than the latest ID existing in `CHANGE_TRACKER_HISTORY`, we don't do anything (as this frequently occurs during checkouts between new and old commits). However, if it is smaller than the latest ID and does not exist in `CHANGE_TRACKER_HISTORY` then we panic.
